tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -95,12 +97,14 @@ export default function CalendarScreen ({ navigation }) {
             let yearNumber = year.toString()
             // If the data for that year doesn't already exist
             if (yearData[yearNumber] === undefined) {
-                newData = GETYearData(year)
-                
+                let newData = GETYearData(year)
```

Comment:
I feel like `await` should go here since `GETYearData` is `async`, but if year scroll works without it don't bother. If you do make this `await` though, you'll have to put the entire function in an async function, then call it.
```javascript
  useEffect(() => {
      async function fetchData() {
       ....
       let newData = await GETYearData(year);
       ....
      }
      fetchData()
    }
  }, [])
```